posts/2024-04-15_gno-tokyo/README.md

```diff
@@ -0,0 +1,65 @@
+---
+publication_date: 2024-04-15T00:00:00Z
+slug: gno-tokyo
+tags: [meetup, tokyo, community]
+authors: [leohhhn]
+---
+    
+# Intro to Gno Recap - Tokyo 2024
+
+[![Banner](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/banner.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/banner.png)
+
+A Gno.land meetup, "Intro to Gno", was held in Tokyo, Japan on the 11th of 
+April 2024. Attendees joined the meetup in one of Tokyo's largest centers, 
+Shinjuku City. The meetup was aimed at a local audience, with the slides being
+in Japanese. 
+
+The meetup focused on explaining Gno.land's major features:
+- Fully open-source smart-contracting
+- Custom smart contracting language for the future
+- Proof of Contribution
+
+[![points](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/three-points.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/three-points.png)
+#### Brief overview of gno.land in Japanese
+
+Gno.land's DevRel Engineer, Leon, covered these three topics separately, discussing the importance of such 
+features in open-source blockchain systems.
+
+[![leon-poc](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/leon-poc.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/leon-poc.png)
+#### @leohhhn with the PoC explanation slide in Japanese
+The Gno.land ecosystem was also introduced: Leon mentioned some of the major 
```

Comment:
**Our ecosystem of contributors**

We spotlighted the projects and contributors building on Gno.land, and explained how they are directly supporting the growth of the ecosystem from the Adena Wallet, to the blockchain explorer Gnoscan, to the first gaming app, Flippando, and an experiment at building a social app and network on Gno.land.